README.md

```diff
@@ -13,7 +13,12 @@ This software needs docker-compose version 1.24.0 at least.
 In an appropriate directory:
 * git clone git@github.com:silinternational/wecarry-api.git
 * cd wecarry-api
-* cp .env.example .env
+* cp local-example.env local.env
```

Comment:
@forevermatt and I were discussing this a few days ago. If we ever want to add a '*.env' to all '.gitignore' files to protect against accidentally committing secrets, this wouldn't work. I think we usually put the `.example` on the end. I'm not totally sold on the idea, since IDEs use the last extension to determine the file type. So use your best judgment until we establish a standard.